joy2twist/config/joy2twist.yaml

```diff
@@ -0,0 +1,21 @@
+/joy2twist_node:
```

Comment:
It should be better to load parameters inside `<node>` and remove namespace here

Reply:
:heavy_check_mark: